### PR TITLE
Fix event decoding with rv 36

### PR DESCRIPTION
**Pull request by KarishmaBothara**
Added types to support rv36 events.
Closes https://github.com/cennznet/api.js/issues/221

### CODE REVIEW

File: packages/api/test/e2e/api.calls.e2e.ts [hunk 1]
```diff
@@ -60,6 +60,42 @@ describe('e2e api calls', () => {
     await apiV36.disconnect();
   });
 
+  it('Subscribe system events for runtime version 36', async done => {
+    const provider = 'wss://cennznet.unfrastructure.io/public/ws';
```
Comment (holygits) [inline packages/api/test/e2e/api.calls.e2e.ts:64]: can we use archive node here?

**Review by holygits — APPROVED** [review]
lgtm

File: packages/api/test/e2e/api.calls.e2e.ts (hunk 1 above)
Comment (KarishmaBothara) [reply packages/api/test/e2e/api.calls.e2e.ts:64]: yes

File: packages/types/src/interfaces/system/types.ts [hunk 2]
```diff
@@ -1,17 +1,40 @@
...
+  readonly isOperational: boolean;
+}
+
+/** @name DispatchInfoTo36 */
+export interface DispatchInfoTo36 extends Struct {
+  readonly weight: WeightTo36;
+  readonly class: DispatchClassTo36;
+  readonly paysFee: bool;
+}
+
```
Comment (alexsednz) [inline packages/types/src/interfaces/system/types.ts:23]: Is the above code auto-generated?

Comment (KarishmaBothara) [reply packages/types/src/interfaces/system/types.ts:23]: yes..

**Pull request merged**